source/managing_account/02_generating_api_key/generating_api_key.rst

```diff
@@ -0,0 +1,30 @@
+Generating an API key
+=====================
+
+From the API keys tab of their account, authenticated users have the possibility to generate API keys, as well as to manage them.
+
+These API keys allow to use the following consultation APIs: ODS Search API v1 and v2, OData, WFS and CSW (more info in the :doc:`Using APIs </exploring_catalog_and_datasets/04_getting_involved/using_api>` documentation).
+
+.. image:: images/account_apikey.png
+
+Click |icon-copypaste| to copy an API key.
+
+Generating an API key
+---------------------
+
+1. Click on the Generate new key button. An API key card appears on the tab.
+2. Write a name for the API key in the Label textbox.
```

Comment:
I could worth it to explain that the name is used to remember what is the purpose of the API key.